components/Chips/tests/unit/ChipsTests.m

```diff
@@ -394,4 +411,40 @@ - (void)testChipsWithDeleteEnabled {
   XCTAssertEqual(controlViewCount, (NSUInteger)1);
 }
 
+- (void)testChipViewDynamicTypeBehavior {
+  // Given
+  MDCChipsTestsFakeChipView *chipView = [[MDCChipsTestsFakeChipView alloc] init];
+  UIFont *titleFont = [UIFont systemFontOfSize:14.0 weight:UIFontWeightMedium];
+  MDCFontScaler *fontScaler = [[MDCFontScaler alloc] initForMaterialTextStyle:MDCTextStyleBody2];
+  titleFont = [fontScaler scaledFontWithFont:titleFont];
+  titleFont = [titleFont mdc_scaledFontAtDefaultSize];
+  chipView.titleFont = titleFont;
+  chipView.mdc_adjustsFontForContentSizeCategory = YES;
+  chipView.mdc_legacyFontScaling = NO;
+  chipView.titleLabel.text = @"Chip";
+  [NSNotificationCenter.defaultCenter
+      postNotificationName:UIContentSizeCategoryDidChangeNotification
+                    object:nil];
+  CGFloat defaultFontSize = chipView.titleLabel.font.pointSize;
+
+  // When
+  if (@available(iOS 10.0, *)) {
+    UIContentSizeCategory size = UIContentSizeCategoryExtraExtraLarge;
+    UITraitCollection *traitCollection =
+        [UITraitCollection traitCollectionWithPreferredContentSizeCategory:size];
+    chipView.traitCollectionOverride = traitCollection;
+  }
+  [NSNotificationCenter.defaultCenter
+      postNotificationName:UIContentSizeCategoryDidChangeNotification
+                    object:nil];
+  CGFloat fontSizeForExtraExtraLargeSizeCategory = chipView.titleLabel.font.pointSize;
```

Comment:
it makes more sense to me if we called this the actualFontSIze

Reply:
Addressed in 865300029f6556aaf570620546e5bcb94e64d6f6